package.json

```diff
@@ -60,6 +61,7 @@
     "chronoshift": "0.8.1",
     "clipboard": "1.7.1",
     "compression": "1.7.1",
+    "core-js": "^2.6.0",
```

Comment:
Please use exact versions. Consider adding this to your npm config:
`npm config set save-exact true`